angularjs-1.5-basics/src/app/interval-example/_interval-example.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+angular.module('interval-example', []);
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->